apidoc/Titanium/UI/Android/Toolbar.yml

```diff
@@ -0,0 +1,255 @@
+    description: |
+        This property is Numberended to be used when the toolbar is used as an ActionBar. It allows the user to specify
+        that the toolbar's background show through the status bar on Android Lollipop and above. The toolbar must be
+        placed at the top of the screen just below the status bar for this property to work as expected. This **must**
+        be specified at creation.
+    type: Boolean
+    default: Undefined. Behaves as if set to false.
+    osver: {android: {min: "21"}}
+    availability: creation
+    
+  - name: translucent
+    summary: If `true`, a translucent background color is used for the toolbar.
+    type: Boolean
+
+methods:
+  - name: collapseActionViews
+    summary: Collapses expanded ActionViews if there is any
+
+  - name: dismissPopupMenus
+    summary: Collapses expandend ActionViews and hides overflow menu
+
+  - name: getContentInsetEnd
+    summary: Returns the margin at the toolbar's content end.
+    returns:
+        type: Number
+
+  - name: getContentInsetEndWithActions
+    summary: Returns the margin after the toolbar's content when there are action buttons.
+    retruns:
+        type: Number
+
+  - name: getContentInsetLeft
+    summary: Returns the margin on the left of the toolbar's content.
+    returns:
+        type: Number
+
+  - name: getContentInsetRight
+    summary: Returns the margin on the right of the toolbar's content.
+    returns:
+        type: Number
+
+  - name: getContentInsetStart
+    summary: Returns the margin at the toolbar's content start. 
+    returns:
+        type: Number
+
+  - name: getContentInsetStartWithNavigation
+    summary: Returns the margin at the toolbar's content start when there is a navigation button.
+    returns:
+        type: Number
+
+  - name: getCurrentContentInsetEnd
+    summary: Returns the margin at the toolbar's content end that will be used with the current configuration of the toolbar.
+    returns:
+        type: Number
+
+  - name: getCurrentContentInsetLeft
+    summary: Returns the margin on the left of the toolbar's content that will be used with the current configuration of the toolbar.
+    returns:
+        type: Number
+
+  - name: getCurrentContentInsetRight
+    summary: Returns the margin on the right of the toolbar's content that will be used with the current configuration of the toolbar.
+    returns:
+        type: Number
+
+  - name: getCurrentContentInsetStart
+    summary: Returns the margin at the toolbar's content start that will be used with the current configuration of the toolbar. 
+    returns:
+        type: Number
+
+  - name: getLogo
+    summary: Returns the source of the logo image in the format it was set.
+
+  - name: getNavigationIcon
+    summary: Returns the source of the navigation image in the format it was set.
+
+  - name: getOverflowIcon
+    summary: Returns the source of the overflow menu image in the format it was set.
```

Comment:
If we document them as properties that would mean that you can access them directly and the widget is listening for their change. This is not the case with the Logo, OverflowIcon, NavigationIcon. They can be changed only through these methods. We can change them to be properties (also the content inset ones) if that will be more convenient from user stand point.